src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer firstBulldozer = new Bulldozer("First Bulldozer", "white", 2020);

```

Comment:
The variables firstBulldozer, secondBulldozer, and thirdBulldozer should be declared final, as they represent constant values and are not modified.